README.md

```diff
@@ -1,11 +1,16 @@
 # funcky-money
 
 [![Build](https://github.com/polyadic/funcky-money/workflows/Build/badge.svg)](https://github.com/polyadic/funcky-money/actions?query=workflow%3ABuild)
+[![Licence: MIT](https://img.shields.io/badge/licence-MIT-green)](https://raw.githubusercontent.com/polyadic/funcky/master/LICENSE-MIT)
 
 Funcky.Money is an implementation of a versatile Money concept which addresses a lot of the problems you encounter when working with money quantities and currencies.
 
 The implementation uses a lot of the concepts from the TDD implementation of Kent Beck in his book: Test-Driven Development by Example.
 
+## Package
+
+[![NuGet package](https://buildstats.info/nuget/Funcky)](https://www.nuget.org/packages/Funcky.Money)
```

Comment:
Don't we usually group this badge with the other badges?

Reply:
No, it's a straight copy from https://github.com/polyadic/funcky